src/DataBase/Models/UserModel.cs

```diff
@@ -10,7 +10,8 @@ public class UserModel
         public string Email { get; set; }
         public string Password { get; set; }
         public string Salt { get; set; }
-
+        [StringLength(3)]
+        public string Currency { get; set; }
```

Comment:
The question that I'm sure was answered but to make it sure: migration files are generated automatically and we should not pay attention to them?